devops/vaults.env

```diff
@@ -35,7 +16,7 @@ STATUS_API_URL="op://API/$APP_ENV/status-api/STATUS_API_URL"
 STATUS_API_VERSION="op://API/$APP_ENV/status-api/STATUS_API_VERSION"
 
 #vaults launchdarkly
-BCREGISTRY_LD_CLIENT_ID="op://launchdarkly/$APP_ENV/bcregistry/BCREGISTRY_LD_CLIENT_ID"
+NAMEEXAMINATION_LD_CLIENT_ID="op://launchdarkly/$APP_ENV/bcregistry/"
```

Comment:
Should this specify which key in bcregistry vault to use?

Reply:
purposefully not specified until we setup a project in LD

Reply:
Ok